src/util/languageUtils.js

```diff
@@ -38,16 +38,10 @@ export const getLanguage = (
 	request,
 	supportedLangs,
 	defaultLang = LANG_DEFAULT
-) => {
-	// return the first language hit in the order of preference
-	supportedLangs.sort(l => l !== defaultLang);
```

Comment:
this is definitely a good place to insert any other sorting rules that are relevant, but at the moment I think we only care that the `defaultLang` is first in the priority list? We could break it out into a function called `sortByPreference` or something if we need to be more specific.

> Perhaps the solution is to document the language list and make clear the implications of language order.

Your original suggestion to document the 'supported languages' setting and make it the responsibility of the caller to submit them in preference order might be the best long-term option. It's a slightly larger refactor, though, so maybe the best short term resolution is to fix the sort function definition and open a ticket to require that `supportedLangs` be provided in order of language preference, default lang first.